app/routes/steps/step2.tsx

```diff
@@ -0,0 +1,10 @@
+import { Link } from "@remix-run/react";
+
```

Comment:
@frederike-ramin Just a suggestion: maybe also add a loader here like in step1 and just show the "street" on the page. For an impressive demo that the value of street is really persisted between those 2 pages.